src/tests/ftest/deployment/server_rank_failure.py

```diff
@@ -131,19 +132,52 @@ def verify_ior_worked(self, ior_results, job_num, errors):
             ior_error = ior_results[job_num][1]
             errors.append("Error found in IOR job {}! {}".format(job_num, ior_error))
 
+    def get_io_inflow(self, engine_kill_host):
```

Comment:
Either here or future, could we make this function more generic and move to a util file or test base? For example (untested):
```
def get_aggregate_metrics(self, metrics=None):
    """Get aggregate metrics data.

    Args:
        metrics (list, optional): list of specific pool metrics to aggregate.

    Returns:
        dict: mapping of metrics to hosts to ranks to aggregate values.
            I.e. {metric: {host: {rank: sum}}}
    """
    agg = {}
    telemetry_data = self.telemetry.get_pool_metrics(metrics)
    for metric, metric_data in telemetry_data.items():
        agg[metric] = {}
        for host, host_data in metric_data.items():
            agg[metric][host] = {}
            for rank, rank_data in host_data.items():
                agg[metric][host][rank] = sum(rank_data.values())

    return agg

# Sample use
agg_metrics = self.get_aggregate_metrics(['engine_pool_ops_fetch'])
for rank, total in agg['engine_pool_ops_fetch'][engine_kill_host].items():
    # do stuff

Reply:
It worked out of the box. Thanks!